test/units/length_test.rb

```diff
@@ -13,6 +13,11 @@ class Measured::LengthTest < ActiveSupport::TestCase
     assert_equal "length", Measured::Length.name
   end
 
+  test "Measured::Length() delegates automatically to .new" do
+    assert_equal Measured::Length.new(1, :in), Measured::Length(1, :in)
+    assert_equal Measured::Length.new(200, :mm), Measured::Length(20, :cm)
```

Comment:
A not yet defined yet call will fall through to `super` and will raise all the way up.

```ruby
[1] pry(main)> Measured::Nothing(1, :cm)
NoMethodError: undefined method `Nothing' for Measured:Module
from /Users/kevin/source/measured/lib/measured/base.rb:21:in `method_missing'
```